cluster/gce/gci/configure-helper.sh

```diff
@@ -161,7 +161,7 @@ function main() {
     else
       if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "tp" ]]; then
         if [[ -z "${DISABLE_NETWORK_SERVICE_SUPPORT:-}" ]]; then
-          start-arktos-network-controller ${KUBERNETES_MASTER_NAME}
+          start-arktos-network-controller ${CLUSTER_NAME}
```

Comment:
Can you give an example of the value for KUBERNETES_MASTER_NAME and CLUSTER_NAME? In local cluster, host IP is used as salt.